go.mod

```diff
@@ -15,8 +15,7 @@ replace (
 require (
 	github.com/blang/semver/v4 v4.0.0
 	github.com/cilium/charts v0.0.0-20240315160837-a5bec4908d75
-	github.com/cilium/cilium v1.16.0-pre.0
-	github.com/cilium/hubble v0.13.2
+	github.com/cilium/cilium v1.16.0-pre.0.0.20240328232606-9ea9af7d936a
```

Comment:
@tklauser i'm in a bit of a chicken and egg situation here 🐔🥚. i need the change in `api/api.go` in https://github.com/cilium/cilium/pull/31077, and https://github.com/cilium/cilium/pull/31077 is the pull request that adds `github.com/cilium/cilium/pkg/cilium-cli/connectivity` package. 

one way to deal with it is to:

- merge this pull request, although it depends on `github.com/cilium/cilium` version that hasn't been merged to main.
- update https://github.com/cilium/cilium/pull/31077 to vendor `github.com/cilium/cilium-cli` main branch to pick up the new `api/api.go`.
- once https://github.com/cilium/cilium/pull/31077 gets merged, update this line to pick up the latest `github.com/cilium/cilium` from the main branch.

it doesn't feel very clean, but i can't think of a better way to deal with this. any thought? 💭

Reply:
Hm, that's a tricky one. I don't think there's a way around depending on a not-merged version in one or the other repo. I guess it will only be present for a short time frame and won't happen again unless we move other packages from cilium/cilium-cli to cilium/cilium.